server/benchmark/models.py

```diff
@@ -20,7 +20,7 @@ class Benchmark(models.Model):
     description = models.CharField(max_length=100, blank=True)
     docs_url = models.CharField(max_length=100, blank=True)
     owner = models.ForeignKey(User, on_delete=models.PROTECT)
-    demo_dataset_tarball_url = models.CharField(max_length=256, blank=True)
+    demo_dataset_tarball_url = models.CharField(max_length=256)
```

Comment:
I remember we talked about this before, but can you remind me how we're getting past situations where demo datasets can't be shared directly with a link?

Reply:
currently we support using synapse storage (e..g, Verena used a private demo dataset)
we should eventually support things like dropbox or google drive